### ISSUE TITLE
Add multimodal support

### ISSUE THREAD

**Issue by joaomdmoura** [joaomdmoura/crewAI#114]
Add the ability to video and audio, though both local and close models

**Comment by m-da-costa** [comment]
Sorry, but what about llava and bakllava?

**Comment by joaomdmoura** [comment]
Yup! I haven't tried yet but those should work, I just a better dsl to support them for real

**Comment by babycommando** [comment]
can't wait for multimodal support!
